schemas/gdbots/ncr/event/labels-updated/latest.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<pbj-schema xmlns="http://gdbots.io/pbj/xsd"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://gdbots.io/pbj/xsd http://gdbots.io/pbj/xsd/schema.xsd">
+
+  <schema id="pbj:gdbots:ncr:event:labels-updated:1-0-0">
+    <fields>
+      <field name="labels_updated" type="string" rule="list" required="true"/>
```

Comment:
@rsumilang labels_added instead of updated for the field name.  rule is set, not required.